### PR TITLE
SEO-Blazor Signature External Linking 

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @ChristoIssac ,

Please review the External Linking changes for the Blazor Signature.

Regards,
Sharly A.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,4 +12,67 @@ To run this application, you need to first clone or download the ‘create a sig
 
 ## Further help
 
-To get more help, check the [ASP.NET Core Blazor documentation](https://docs.microsoft.com/en-us/aspnet/core/blazor).
+To get more help, check the [ASP.NET Core Blazor documentation](https://docs.microsoft.com/en-us/aspnet/core/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-signature-github-samples).
```
Comment (Girija-Rajendran) [inline README.md]: remove UTM param here

File: README.md [hunk 2]
```diff
@@ -19,3 +13,66 @@ To run this application, you need to first clone or download the ‘create a sig
 ## Further help
 
 To get more help, check the [ASP.NET Core Blazor documentation](https://docs.microsoft.com/en-us/aspnet/core/blazor).
+
+## Features and Benefits
+
+### Drawing
+
+[Draw](https://blazor.syncfusion.com/documentation/signature/draw?utm_source=github&utm_medium=listing&utm_campaign=blazor-signature-github-samples)the given text as a signature using different font families and font sizes.
```
Comment (Girija-Rajendran) [inline README.md]: add space between words
image

File: README.md [hunk 3]
```diff
@@ -19,3 +13,66 @@ To run this application, you need to first clone or download the ‘create a sig
...
+
+The Signature Pad component supports various [customization](https://blazor.syncfusion.com/documentation/signature/customization?utm_source=github&utm_medium=listing&utm_campaign=blazor-signature-github-samples) options like background color, background image, stroke color, stroke width, save with background, undo, redo, clear, readonly, and disabled.
+
+### Loading
+
+It supports to load a pre drawn signature to component either as base 64 or URL. It supports PNG, JPEG, and SVG image types.
+
+### User Interaction
+
+The Signature component supports various interaction like Undo, Redo, Clear, Disabled, and ReadOnly and Every changes occurred in signature can be taken as a snap and saved to collection for handling the above
```
Comment (Girija-Rajendran) [inline README.md]: image
image

File: README.md (hunk 2 above)
Comment (Sharly-Atieno) [reply README.md]: changes resolved

File: README.md (hunk 3 above)
Comment (Sharly-Atieno) [reply README.md]: changes resolved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**